src/vma/proto/vma_lwip.cpp

```diff
@@ -133,6 +133,7 @@ vma_lwip::vma_lwip()
 		free_lwip_resources();
 		throw_vma_exception("LWIP: failed to register timer event");
 	}
+	safe_mce_sys().sysctl_reader.get_tcp_keepalive_info(true);
```

Comment:
please consider do initialization in common place as https://github.com/Mellanox/libxlio/blob/master/src/core/util/sysctl_reader.h#L95

Reply:
Sounds good!